spec/javascripts/components/reorderable-list-spec.js

```diff
@@ -0,0 +1,206 @@
+/* eslint-env jasmine, jquery */
+/* global GOVUK, spyOnEvent */
```

Comment:
I think this can be simplified to `/* global spyOnEvent */` as we did in collections publisher.

Reply:
done

Reply:
I reverted this as it fails tests:

```
$ standardx 'app/assets/javascripts/**/*.js' 'spec/javascripts/**/*.js'
standardx: Use JavaScript Standard Style (tweaked by standardx) (https://github.com/standard/standardx)
  /var/lib/jenkins/workspace/onents_add-reorderable-component/spec/javascripts/components/reorderable-list-spec.js:56:29: 'GOVUK' is not defined.
  /var/lib/jenkins/workspace/onents_add-reorderable-component/spec/javascripts/components/reorderable-list-spec.js:89:29: 'GOVUK' is not defined.
  /var/lib/jenkins/workspace/onents_add-reorderable-component/spec/javascripts/components/reorderable-list-spec.js:120:29: 'GOVUK' is not defined.
  /var/lib/jenkins/workspace/onents_add-reorderable-component/spec/javascripts/components/reorderable-list-spec.js:151:29: 'GOVUK' is not defined.
  /var/lib/jenkins/workspace/onents_add-reorderable-component/spec/javascripts/components/reorderable-list-spec.js:181:29: 'GOVUK' is not defined.
error Command failed with exit code 1.
```

It's cleaner to have it there than to do `window.GOVUK`